src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -1249,7 +1259,7 @@ public ResultSet getExportedKeys(String catalog, String schema, String table)
 
         catalog = (catalog != null) ? quote(catalog) : null;
 
-        String quotedSchema = (schema != null) ? quote(schema) : null;
+        String quotedSchema = (schema != null) ? quote(schema) : quote("main");
```

Comment:
similar as above, we should return the actual schema of the retrieved PK's table, not the parameter that was passed to the function